org.thonny.Thonny.yaml

```diff
@@ -0,0 +1,314 @@
+---
+id: org.thonny.Thonny
+branch: stable
+runtime: org.freedesktop.Sdk
+runtime-version: "20.08"
+sdk: org.freedesktop.Sdk
+command: thonny
+cleanup:
+  - /lib/debug
```

Comment:
```suggestion
```

Debug info is split out in it's own package, don't remove it.

Reply:
My bad, I was working around flatpak/flatpak#469 but I can workaround this in a better way.